src/PrestaShopBundle/Routing/Linter/LinterException.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Routing\Linter;
+
+use RuntimeException;
+
+/**
+ * Is thrown when error occurs during linting.
+ */
+class LinterException extends RuntimeException
```

Comment:
Not sure if this is useful 🤔we never intend to catch these, right ?

Reply:
It depends, you may catch and log them or you can fail as soon as first exception is thrown.

Reply:
OK but `LinterException` is not accurate then 🤔 it would be rather `BadConfigurationException`, `BadlyConfiguredControllerException` or `MissingSecurityException`

Reply:
Well, I think it's ok for now, if we add more linters, we can make these exceptions more specific, wdyt?

Reply:
All right but `LinterException` really does not make sense to me 😄